public/book/pages/1/1.txt

```diff
@@ -1,5 +1,5 @@
 A green and yellow parrot, which hung in a cage outside the door, kept repeating over and over:
-“Allez vous-en! Allez vous-en! Sapristi! That’s all right!”
+“Allez&frvous-en!&frAllez&frvous-en!&frSapristi! That’s all right!”
```

Comment:
Be careful with preceding spaces. I would expect this line to look like:

`“Allez&frvous-en! Allez&frvous-en! Sapristi! That’s all right!”`

Reply:
I'm not sure if it makes a difference, but this is actually one phrase. Some of them have weird formats

![french](https://user-images.githubusercontent.com/72589629/163731745-04d19df1-e654-4af6-83c5-608c08e076c4.png)